README.md

```diff
@@ -276,6 +277,42 @@ client.links.getInfo({ url: "https://www.jet.com" })
   .catch(handleError);
 ```
 
+### Transactions
```

Comment:
Reading this documentation I've be confused about the "accounts" scoped transactions and this top-level resource. Wdyt about adding a bit of prose here describing when you'd use either?

Reply:
Done